linodecli/plugins/get-kubeconfig.py

```diff
@@ -0,0 +1,170 @@
+"""
+This plugin allows the option to merge an LKE Cluster's kubeconfig to a desired location.
+
+Usage:
+
+   linode-cli get-kubeconfig --id <cluster-id> --kubeconfig /path/to/config/file
+"""
+
+import argparse
+import base64
+import os
+import sys
+
+import yaml
+
+PLUGIN_BASE = "linode-cli get-kubeconfig"
+
+
+def call(args, context):
+    """
+    The entrypoint for this plugin
+    """
+    parser = argparse.ArgumentParser(PLUGIN_BASE, add_help=True)
+
+    group = parser.add_mutually_exclusive_group()
+
+    parser.add_argument(
+        "--kubeconfig",
+        metavar="KUBECONFIG",
+        nargs="?",
+        help="Path to kubeconfig file. If omitted, ~/.kube/config will be used.",
+        default="~/.kube/config",
+    )
+    parser.add_argument(
+        "--dry-run",
+        default=False,
+        const=True,
+        nargs="?",
+        help="Print resulting merged kubeconfig instead of merging.",
+    )
+    group.add_argument(
+        "--label",
+        metavar="LABEL",
+        nargs="?",
+        help="Label for desired cluster. If omitted, the ID must be provided.",
+    )
+    group.add_argument(
+        "--id",
+        metavar="ID",
+        nargs="?",
+        help="ID for desired cluster. If omitted, the Label must be provided.",
+    )
+
+    parsed = parser.parse_args(args)
+
+    # If --id was used, fetch the kubeconfig using the provided id
+    if parsed.id:
+        code, kubeconfig = context.client.call_operation(
+            "lke", "kubeconfig-view", args=[parsed.id]
+        )
+
+        if code != 200:
+            print(f"Error retrieving kubeconfig: {code}", file=sys.stderr)
+            sys.exit(1)
+
+    # If --label was used, fetch the kubeconfig using the provided label
+    elif parsed.label:
+        kubeconfig = _get_kubeconfig_by_label(parsed.label, context.client)
+    else:
+        print("Either --label or --id must be used.", file=sys.stderr)
+        sys.exit(1)
+
+    # Load the specified cluster's kubeconfig and the current kubeconfig
+    clusterConfig = yaml.safe_load(
+        base64.b64decode(kubeconfig["kubeconfig"]).decode()
+    )
+    currentConfig = None
+
+    if os.path.exists(os.path.expanduser(parsed.kubeconfig)):
+        currentConfig = _load_config(os.path.expanduser(parsed.kubeconfig))
+
+    # If there is no current kubeconfig, dump the cluster config to the specified file location.
+    # If there is a current kubeconfig, merge it with the cluster's kubeconfig
+    if not currentConfig:
```

Comment:
This could be simplified to something like:
```python
clusterConfig = _merge_dict(currentConfig, clusterConfig) if currentConfig is not None else clusterConfig

if parsed.dry_run:
    ....
else:
    ....
```